### PR TITLE
do verbose logging unless its a linux console

### PR DESCRIPTION
**Pull request by jakemac53**
This makes atom/intellij consoles work as expected

### CODE REVIEW

**Comment by kevmoo** [comment]
Different on mac?

**Comment by jakemac53** [comment]
by linux I really meant !windows. I haven't actually tried on a mac console but it the condensed logger should work.

File: lib/src/generate/options.dart [hunk 1]
```diff
@@ -90,7 +90,9 @@ class BuildOptions {
...
 
-final _cyan = Platform.isWindows ? '' : '\u001b[36m';
-final _yellow = Platform.isWindows ? '' : '\u001b[33m';
-final _red = Platform.isWindows ? '' : '\u001b[31m';
-final _endColor = Platform.isWindows ? '' : '\u001b[0m';
+final _cyan = _isLinuxConsole ? '\u001b[36m': '';
+final _yellow = _isLinuxConsole ? '\u001b[33m' : '';
+final _red = _isLinuxConsole ? '\u001b[31m' : '';
+final _endColor = _isLinuxConsole ? '\u001b[0m' : '';
+final _isLinuxConsole =
```
Comment (kevmoo) [inline lib/src/generate/options.dart]: ... `_isPosixConsole`?

**Comment by kevmoo** [comment]
otherwise, LGTM ...or `_isPosixTerminal`

**Pull request merged**